### ISSUE TITLE
Flatten contract source using CLI

### ISSUE THREAD

**Issue by quintinbrooks** [eth-brownie/brownie#849]
### Overview
_Currently, Brownie doesn't provide a way to flattening smart contracts._ Feature for flattening smart contracts can be a way for testing contracts with Brownie, while deploying them using another tool. Also, it can simplify the tasks required for verifying an smart contract on the Etherscan.

### Specification
For the CLI following simple argument can be used:
```
brownie --flatten
```

### Dependencies
#411 is related to this issue. Because, probably flattening smart contracts (written in Solidity) will make it easier to verify them on Etherscan.

**Comment by iamdefinitelyahuman** [comment]
This could be useful as a flag within `brownie compile`, I think.

**Comment by matnad** [comment]
Implemented by #914. 
@quintinbrooks  Use `ContractContainer.get_verification_info()["flattened_source"]`

**Comment by nazariyv** [comment]
@matnad looks like 'flattened_source' is no longer available in 1.17.0?

**Comment by DanielVF** [comment]
```python
container = YourContractName
print("\n\n".join([x['content'] for x in container.get_verification_info()['standard_json_input']['sources'].values()]))
```
